src/Microsoft.AspNetCore.Authentication.OAuth/JsonClaimMapperCollectionExtensions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public static class JsonClaimMapperCollectionExtensions
```

Comment:
I'm not sure I like this as extension methods on an IEnumerable.

What do you think about making the public API into a ClaimsManager instead, with similar API?  The property is then just named ClaimsManager.

```C#
   ClaimsManager.CopyJsonKey(ClaimsTypes.NameIdentifier, "id");
   ClaimsManager.Delete(ClaimsTypes.ReallyLongStuffIDon'tWant);
```

You can still have these methods add some kind of strongly typed Action that maps to the Map classes, or directly add an Action with the signature of Map.  I don't think the extensiblity changes much as they could still add extension methods to the Manager just like the collection.